requirements.txt

```diff
@@ -1,6 +1,7 @@
 fasttext
 GitPython
-hyperscan==0.1.5
+hyperscan==0.2.0; sys.platform == "dawin"
```

Comment:
@marcorosa typo: `darwing` instead of "dawin"

Reply:
yes, thanks!